test/integration/scheduler_perf/scheduler_bench_test.go

```diff
@@ -45,15 +49,15 @@ func BenchmarkScheduling(b *testing.B) {
 	for _, test := range tests {
 		name := fmt.Sprintf("%vNodes/%vPods", test.nodes, test.existingPods)
 		b.Run(name, func(b *testing.B) {
-			benchmarkScheduling(test.nodes, test.existingPods, test.minPods, setupStrategy, testStrategy, b)
+			benchmarkScheduling(test.nodes, test.existingPods, test.minPods, defaultNodeStrategy, setupStrategy, testStrategy, b)
 		})
 	}
 }
 
-// BenchmarkSchedulingAntiAffinity benchmarks the scheduling rate of pods with
+// BenchmarkSchedulingPodAntiAffinity benchmarks the scheduling rate of pods with
 // PodAntiAffinity rules when the cluster has various quantities of nodes and
 // scheduled pods.
-func BenchmarkSchedulingAntiAffinity(b *testing.B) {
+func BenchmarkSchedulingPodAntiAffinity(b *testing.B) {
```

Comment:
Sure. From below snapshot, it's very likely it's running `-bench=.` option:

![image](https://user-images.githubusercontent.com/1425903/47125165-b9149180-d236-11e8-95f5-2c36914eb44d.png)

@wojtek-t @shyamjvs If function `BenchmarkSchedulingAntiAffinity` is renamed to  `BenchmarkSchedulingPodAntiAffinity`, I guess old datapoints would stop growing and become stale? And maybe we need to do some trick (manually rename the local dataset file) to make old data connect with new data, so that the graph is still consistent.

Reply:
I'm not familiar with perf-dash.
@krzysied - can you please suggest something?

Reply:
> I guess old datapoints would stop growing and become stale

Yes. The new datapoints will by assigned to new function name. The old one will remain unchanged.

> And maybe we need to do some trick (manually rename the local dataset file) to make old data connect with new data, so that the graph is still consistent.

Can be done. The question is whether we need this.
Regarding benchmarks, Perfdash presents 100 newest results. Assuming that there is one test every hour, after a week there will be only results with the new function name available.

Reply:
Thanks @krzysied @wojtek-t , then don't bother to "migrate" old dataset :)